src/vs/editor/contrib/suggest/browser/suggestWidget.ts

```diff
@@ -1041,3 +1044,37 @@ export class SuggestContentWidget implements IContentWidget {
 		this._position = position;
 	}
 }
+
+function getLabelForKind(kind: CompletionItemKind): string {
```

Comment:
This should be living here: https://github.com/microsoft/vscode/blob/c1fb4c3983bcc28e63dd159dbbb2ea98d352d33e/src/vs/editor/common/languages.ts#L425